live-examples/js-examples/atomics-xor.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">// create a SharedArrayBuffer with a size in bytes
+var buffer = new SharedArrayBuffer(16);
+var uint8 = new Uint8Array(buffer);
+uint8[0] = 7;
+
+Atomics.xor(uint8, 0, 2); // (returns 7)
+
+// 7 (0111) XOR 2 (0010) = 7 (0101)
```

Comment:
Should be:

`// 7 (0111) XOR 2 (0010) = 5 (0101)`

Reply:
Yes. Good spot.